alphadia/workflow/manager.py

```diff
@@ -483,6 +483,17 @@ def fit_predict(self, update_dict):
 
 
 class FDRManager(BaseManager):
+    """
+    Contains, updates and applies classifiers for target-decoy competitio-based false discovery rate (FDR) estimation.
+    --------------------------------
```

Comment:
We ususally add the init docstring below the init `__init__`.
Can you also move the `----` below `Parameters` ?